### PR TITLE
Add map to web page

### PR DESCRIPTION
**Pull request by panerorenn9541**
-Map takes up the entire screen
-Map begins zoomed in on Googleplex building
-Map API key is hidden in separate file

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -11,3 +11,19 @@
...
 // See the License for the specific language governing permissions and
 // limitations under the License.
+ 
+//Get API key from hidden file and use it to get the map
+var mykey = keys.MAPS_API_KEY;
+document.getElementById('mapUrl').src = mykey;
+ 
+//When the page loads, call createMap
+window.onload = function() {
+  createMap()
```
Comment (kinsaurralde) [inline src/main/webapp/script.js]: semicolon

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -11,3 +11,19 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+ 
+//Get API key from hidden file and use it to get the map
```
Comment (kinsaurralde) [inline src/main/webapp/script.js]: I think there should be a space between // and comment text

**Review by kinsaurralde — APPROVED** [review]

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
 <html>
-
+
```
Comment (huzcn) [inline src/main/webapp/index.html]: remove empty space

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -1,15 +1,17 @@
...
   <link rel="stylesheet" href="style.css">
+  <script type="text/javascript" src="apikeys.js"></script>
+  <script id="mapUrl"></script>
   <script defer src="script.js"></script>
 </head>
-
-<body>
-
+ 
+<body>
```
Comment (huzcn) [inline src/main/webapp/index.html:13]: remove empty spaces.

File: src/main/webapp/index.html [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
-<body>
-
+ 
+<body>   
+  <div id="map"></div>
 </body>
-
-</html>
\ No newline at end of file
+
```
Comment (huzcn) [inline src/main/webapp/index.html]: here as well

File: src/main/webapp/index.html (hunk 4 above)
Comment (huzcn) [inline src/main/webapp/index.html]: remove empty spaces.

**Review by huzcn — APPROVED** [review]
Approve after nits are resolved.

**Comment by panerorenn9541** [comment]
Resolve all issues

**Pull request merged**